main.go

```diff
@@ -277,21 +277,24 @@ func main() {
 		return
 	}
 
-	originPath := flag.Arg(0)
+	originPaths := flag.Args()
 
 	if filePath != "" {
-		deprecatedMessagesCh <- fmt.Sprintf("-%s is deprecated. Put file name as last argument to the command(Example: goimports-reviser -rm-unused -set-alias -format goimports-reviser/main.go)", filePathArg)
-		originPath = filePath
+		deprecatedMessagesCh <- fmt.Sprintf("-%s is deprecated. Put file name(s) as last argument to the command(Example: goimports-reviser -rm-unused -set-alias -format goimports-reviser/main.go)", filePathArg)
+		originPaths = append(originPaths, filePath)
 	}
 
-	if originPath == "" {
-		originPath = reviser.StandardInput
+	var originPath string
+	if len(originPaths) == 0 || (len(originPaths) == 1 && originPaths[0] == "-") {
+		originPaths[0] = reviser.StandardInput
 	}
 
-	if err := validateRequiredParam(originPath); err != nil {
-		fmt.Printf("%s\n\n", err)
-		printUsage()
-		os.Exit(1)
+	for _, originPath = range originPaths {
+		if err := validateRequiredParam(originPath); err != nil {
+			fmt.Printf("%s\n\n", err)
+			printUsage()
+			os.Exit(1)
```

Comment:
You are looping on argument to validate them.

Maybe you could report an error for each argument

Then after the loop printusage and exit, when at least an error is found

Reply:
Actually, given that `validateRequiredParam` is currently only checking to see if stdin exists in the cast of `originPaths[0]` being `-` I think this entire loop can be removed and the call to `validateRequiredParam` moved up into the previous block. @incu6us I'm going to make this change but let me know what you think?

Reply:
@n-oden yep, agree